### ISSUE TITLE
How to edit?

### ISSUE THREAD

**Issue by fredt34** [timjansen/hanson#9]
Hi,

I tried to edit a .hanson file on my Mac with SublimeText, but the format is not recognized - and if I rename it as .json file, it turns in a garbage of pink words... Can you recommend any editor / trick to edit the file easily?

Thanks for this nice little hack of json, btw!

**Comment by timjansen** [comment]
Yes, the trick is to map the extension .hanson (or .hson, which is what I
am using) to use JavaScript's syntax highlighting.

Tim
